includes/connect/class-wc-stripe-connect.php

```diff
@@ -9,9 +9,6 @@
 	 * Stripe Connect class.
 	 */
 	class WC_Stripe_Connect {
-
-		const SETTINGS_OPTION = 'woocommerce_stripe_settings';
```

Comment:
We shouldn't remove this constant; merchant custom code may depend on it.
We should add a deprecated comment.

Reply:
Reverted it and added a comment above 👍